<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeNicknameDialog</class>
 <widget class="QDialog" name="ChangeNicknameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Change Nickname</string>
  </property>
  <widget class="QTextEdit" name="change_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
